x-pack/plugins/synthetics/server/synthetics_service/formatters/private_formatters/format_synthetics_policy.ts

```diff
@@ -23,6 +23,7 @@ export interface ProcessorFields {
   'monitor.id': string;
   test_run_id: string;
   run_once: boolean;
+  spaceId: string;
```

Comment:
All other properties use snake_case but the new property is camelCase, shouldn't we keep consistency here?